console/ui/src/app/console.service.ts

```diff
@@ -486,6 +525,13 @@ export interface ApiChannelMessage {
   user_id_two?: string
 }
 
+export interface ApiChannelMessageList {
+  messages?: ApiChannelMessage[]
+  next_cursor?: string
+  prev_cursor?: string
+  cacheable_cursor?: string
```

Comment:
this doesn't seem to be used anywhere

Reply:
This is used on `chatMessages.component.ts`

Reply:
There are`next_cursor` and `prev_cursor`, but seems to be no `cacheable_cursor`

Reply:
This message is shared between console API and main Nakama API - the `cacheable_cursor` is used in the clients.